pyproject.toml

```diff
@@ -358,13 +358,13 @@ ignore = [
   "SIM117",  # https://github.com/pyvista/pyvista/pull/5888
   "SIM118",  # https://github.com/pyvista/pyvista/pull/5837
 ]
-unfixable = []
+unfixable = ['F401']
```

Comment:
I want to highlight this change. Unused imports will no longer be auto-fixed (removed). The auto-fix is nice in many instances, but I find it to be a bit aggressive in others. When I'm mid-development (and have unused imports that I plan to use later) I like to use pre-commit to clean up the formatting even though I'm not yet ready to commit. But this always removes the imports when I don't want them removed.